.changeset/fifty-lizards-sink.md

```diff
@@ -0,0 +1,9 @@
+---
+"@fluidframework/ai-collab": minor
+"__section": deprecation
+---
```

Comment:
```suggestion
---
---
"__section": deprecation
---
```

See https://github.com/microsoft/FluidFramework/wiki/Changesets#changesets-that-apply-to-no-packages

Reply:
keep the form "__section"